Fixes https://github.com/compiler-explorer/compiler-explorer/issues/4325
-- 
diff --git a/lib/handlers/compile.ts b/lib/handlers/compile.ts
index c96c9a73..083fe6a1 100644
--- a/lib/handlers/compile.ts
+++ b/lib/handlers/compile.ts
@@ -204,7 +204,8 @@ export class CompileHandler {
                 compiler.exe = exe;
             } else {
                 // errors resolving to absolute path are not fatal for backwards compatibility sake
-                logger.error(`Unable to resolve '${compiler.exe}'`);
+                logger.warn(`Unable to resolve '${compiler.exe}'`);
+                return null;
             }
         }
 
